### ISSUE TITLE
`alias_method': undefined method `column_spec' for class `ActiveRecord::ConnectionAdapters::OracleEnhancedAdapter'

### ISSUE THREAD

**Issue by jscrane** [rsim/oracle-enhanced#746]
I'm not a ruby guy at all but I'm trying to get started with mongify and am seeing this error trying to read from an existing Oracle db.

```
/var/lib/gems/2.1.0/gems/activesupport-3.2.22/lib/active_support/core_ext/module/aliasing.rb:31:in `alias_method': undefined method `column_spec' for class `ActiveRecord::ConnectionAdapters::OracleEnhancedAdapter' (NameError)
	from /var/lib/gems/2.1.0/gems/activesupport-3.2.22/lib/active_support/core_ext/module/aliasing.rb:31:in `alias_method_chain'
	from /var/lib/gems/2.1.0/gems/activerecord-oracle_enhanced-adapter-1.6.5/lib/active_record/connection_adapters/oracle_enhanced/column_dumper.rb:8:in `block in included'
	from /var/lib/gems/2.1.0/gems/activerecord-oracle_enhanced-adapter-1.6.5/lib/active_record/connection_adapters/oracle_enhanced/column_dumper.rb:6:in `class_eval'
```

I'm on Linux Mint, Debian edition, x86_64, with:
- ruby 2.1.5p273
- Rails 4.2.5
- activerecord-oracle_enhanced-adapter-1.6.5
- instantclient_12_1 (also tried instantclient_11_2)
- ruby-oci8-2.2.1

Any ideas?

**Comment by yahonda** [comment]
Your report shows `activesupport-3.2.22`. Looks like it is not `Rails 4.2.5`. Would you validate that you re installing rails 4.2.5.

**Comment by jscrane** [comment]
Yep:
```
$ rails -v
Rails 4.2.5
```

**Comment by yahonda** [comment]
Thanks for the update. Let me rephrase my question. I believe your environment also has Rails 3.2.22 installed. 
```
/var/lib/gems/2.1.0/gems/activesupport-3.2.22/lib/active_support/core_ext/module/aliasing.rb:31:in `alias_method': undefined method `column_spec' for class `ActiveRecord::ConnectionAdapters::OracleEnhancedAdapter' (NameError)
```

Would you execute `bundle show` and uninstall Rails 3.2.22. or other version of Rails except for 4.2.5.

**Comment by jscrane** [comment]
Thank you very much for your help. It seems that mongify pulls in the -3.2.22 versions of things and I compounded my error by installing rails on top.

I wiped my ruby install and installed activerecord-oracle_enhanced-adapter:1.4.0 and it works fine.

Thanks again!

**Comment by yahonda** [comment]
Thanks. Glad you have resolved it.
